src/main/resources/static/templates/home.html

```diff
@@ -89,46 +88,45 @@ <h1>Non blocking IO</h1>
                 <span class="icon"></span>
                 <h1>Efficient Message Passing</h1>
                 <p>
-                    Reactor Operators and Schedulers can <strong>sustain high
-                    throughput rates</strong> on the order of 10's of
-                    millions of messages per second. Plus its <b>low memory footprint</b> should go under
-                    most of the radars. Reactor Core is the first reactive
+                    Reactor operators and schedulers can <strong>sustain high
+                    throughput rates</strong>, on the order of 10's of
+                    millions of messages per second. Its <b>low memory footprint</b> goes under
+                    most radars. Reactor Core is the first reactive
                     library based on the joint reactive <a
                         href="https://github.com/reactor/reactive-streams-commons">research
-                    effort</a> also implemented by <a href="http://reactivex.io">RxJava 2
-                </a>.
+                    effort</a> also implemented by <a href="http://reactivex.io">RxJava 2</a>.
                 </p>
             </div>
         </article>
         <article class="friendly-adoption">
             <div>
-                <h1>Micro Reactive Toolkit for all</h1>
+                <h1>A Micro Reactive Toolkit for All</h1>
                 <p>
-                    <strong> Do not write Reactive Streams yourself!</strong>
-                    Reactor modules are both embeddable and interoperable. They focus on
-                    providing a rich functional
+                    Don't write Reactive Streams yourself!
+                    Reactor's modules are <strong>embeddable and interoperable</strong>. They focus on
+                    providing <strong>rich and functional</strong>
                     <a href="https://www.reactive-streams.org/">Reactive Streams</a>
-                    APIs. Use Reactor at any level of granularity :
+                    APIs. You can use Reactor at any level of granularity:
                     <ul>
-                        <li>In Frameworks such as <a href="https://spring.io">Spring
-                        </a> Boot and WebFlux. </li>
-                        <li>In Drivers such as <a href="https://github.com/cloudfoundry/cf-java-client">CloudFoundry Java Client</a></li>
-                        <li>In Contracts or Protocols such as <a href="http://rsocket.io">RSocket</a> and <a href="https://r2dbc.io">R2DBC</a></li>
+                        <li>in frameworks such as <a href="https://spring.io">Spring
+                        </a> Boot and WebFlux</li>
+                        <li>in drivers such as the <a href="https://github.com/cloudfoundry/cf-java-client">CloudFoundry Java Client</a></li>
```

Comment:
you can actually make it a bit more correct by replacing `drivers` by `drivers and clients` (the example given is more a client than a driver I'd say)